scripts/build_snap.sh

```diff
@@ -7,6 +7,10 @@ if [ ! -d "dvc" ]; then
   exit 1
 fi
 
+# generate bash completion script
+pip install .
+dvc completion -s bash -f completion.sh .
+
```

Comment:
Ideally this should be done within the snapcraft build but haven't worked out how to get at its interpreter/virtual-env during the build process to run python -m dvc completion ....

There aren't any docs about this.

Reply:
nvm managed it by trawling through lots of source code.